libs/domain/user/src/services/routes.ts

```diff
@@ -0,0 +1,16 @@
+import { RouteType } from '@spryker-oryx/router';
+import { RouteConfig } from '@spryker-oryx/router/lit';
+import { html, TemplateResult } from 'lit';
+
+export const userRoutes: RouteConfig[] = [
+  {
+    path: '/account/:page',
+    type: RouteType.Account,
+  },
+  {
+    path: '/account/*',
+    type: RouteType.NotFound,
+    render: (): TemplateResult =>
+      html`<oryx-composition route="/account/:page"></oryx-composition>`,
+  },
```

Comment:
Why do we need this route?

Reply:
It's a fallback route in case you end up at an account page that doesn't exist. As inidicated in the acceptance criteria and e2e acceptance criteria in the ticket.